DuckDuckGo/DuckPlayer/YoutubeOverlayUserScript.swift

```diff
@@ -131,23 +130,22 @@ final class YoutubeOverlayUserScript: NSObject, Subfeature {
         return nil
     }
 
-    // MARK: - UserValuesNotification
-
-    struct UserValuesNotification: Encodable {
-        let userValuesNotification: UserValues
+    private func handleSettingsChange() {
+        let values = UserValues(duckPlayerMode: duckPlayer.settings.mode, askModeOverlayHidden: duckPlayer.settings.askModeOverlayHidden)
+        userValuesUpdated(userValues: values)
     }
     
     deinit {
-        userValuesCancellable.removeAll()
+        cancellables.removeAll()
     }
 }
 
 extension YoutubeOverlayUserScript {
     @MainActor
     func handleSendJSPixel(params: Any, message: UserScriptMessage) -> Encodable? {
-        guard let body = message.messageBody as? [String: Any], let parameters = body["params"] as? [String: Any] else {
-            return nil
-        }
+        // guard let body = message.messageBody as? [String: Any], let parameters = body["params"] as? [String: Any] else {
```

Comment:
Maybe remove this if it's not being used?

Reply:
That's coming up in the next PR when pixels are added